packages/hoppscotch-common/src/pages/index.vue

```diff
@@ -71,13 +71,41 @@
       @hide-modal="onCloseConfirmSaveTab"
       @resolve="onResolveConfirmSaveTab"
     />
-    <HoppSmartConfirmModal
```

Comment:
You should not remove this confirmation modal as this is related to closing all tabs. This will break functionality You should replace the previous confirmation modal (starting at line 67) with the new `HoppSmartModal` implementation

Reply:
It was my mistake. I fixed it